README.md

```diff
@@ -288,6 +288,10 @@ A wide variety of companies and organizations use APISIX for research, productio
 
 Users are encouraged to add themselves to the [Powered By](doc/powered-by.md) page.
 
+## Contributor Over Time
+
+![](./doc/images/contributor-over-time.png)
```

Comment:
We’d better add some descriptions here, so when this image is broken due to network, this description will show properly.

Reply:
nice catch

Reply:
@Miss-you ping